Bottom: fc397e23417bf15770582bf75fc43e352a6a1f09
Top:    c93a3c2293ff4b95f7cd9703ad07bb9241ddcac6
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-07 15:03:04 +0100

Refresh of doc-remove-eps-files-in-favour

---

diff --git a/doc/Makefile.am b/doc/Makefile.am
old mode 100755
new mode 100644
